fixBindgenShim.js

```diff
@@ -113,9 +113,12 @@ export function setRandomSeed(seed) {\n\
 
     //prettier-ignore
     const getRandomValuesReplacementCode =
-    `(randomSeed instanceof Uint8Array && randomSeed.length === 32) ?\n \
-        varg1.set(randomSeed, 0) :\n \
-        getObject(arg0).getRandomValues(varg1); \
+    `if(randomSeed instanceof Uint8Array && randomSeed.length === 32) {\n\
+       varg1.set(randomSeed, 0);\n\
+       randomSeed = undefined;\n\
```

Comment:
Is there any use in setting the `randomSeed` to 0 before setting it to `undefined`?

Reply:
No cause the seed, when set, is a Uint8Array not a number, so either way we're just clearing out that value.

Reply:
So `randomSeed` is pretty much just a pointer to the array? And there's a possibility that someone copied that pointer into some other pointer somewhere, so they could still access the array after you nuked `randomSeed`?